kratos/python_scripts/KratosUnittest.py

```diff
@@ -35,19 +35,19 @@ def run(self, result=None):
         super(TestCase,self).run(result)
 
     def failUnlessEqualWithTolerance(self, first, second, tolerance, msg=None):
-        ''' fails if first and second have a difference greater than
+        ''' Fails if first and second have a difference greater than
         tolerance '''
 
         if first < (second - tolerance) or first > (second + tolerance):
             raise self.failureException(msg or '%r != %r within %r places' % (first, second, tolerance))
 
-    def assertIsClose(self, first, second, rel_tol=None, abs_tol=None, msg=None):
-        """Fail if the two objects are unequal as determined by their
-           absolute and relative difference
+    def failUnlessAbsAndRelDifference(self, first, second, rel_tol=None, abs_tol=None, msg=None):
```

Comment:
why the rename?

Reply:
To be consistent with the rest of the unittest api. They never define the assertion directly with its user name.

Honestly I don't know the reason for that, but as it is an extension of their API I would like to keep it their way.

Reply:
but what abt assertTrue, assertEqual, assertAlmostEqual ... ?
Those are the ones we use most of the time I would say

Reply:
https://docs.python.org/2/library/unittest.html#deprecated-aliases

This. Actually it seems that is only relevant for versions previous to 2.7 . Since we do not have support for that I can change all of them to the assert version.